src/tasks/actions/thunks.ts

```diff
@@ -399,6 +403,9 @@ export const cloneTask = (task: Task) => async (dispatch: Dispatch<Action>) => {
     })
 
     const fluxWithNewName = format_from_js_file(ast)
+    if (!isValidFlux(fluxWithNewName)) {
+      throw new Error('Invalid flux script. Please check your query text.')
```

Comment:
This one is not tested, since it's cloning an existing task which has invalid flux.
Since we can no longer create tasks which are invalid, we cannot create an invalid task to the be cloned.

Instead, this is something the users will encounter in the wild => when they attempt to clone invalid flux tasks.